webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+def test_click_numbers_link(session):
+    # 14.1 Step 7
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href=%s>123456</a>" % url)
```

Comment:
How is a link with numbers diffferent to a string?

Reply:
That's what David asked for. He wanted a test of a link with numbers.

Reply:
I think we’ve moved away from this practice since.

Reply:
The reasoning being obvious: The spec change and suddenly the comments will not be in sync, causing confusion. As with mapping tests to specific lines in algorithms, it’s a battle you cannot win.

Reply:
Should I delete this test?

Reply:
Sorry, I realise I answered the wrong question earlier: when you asked about numbers I mistakenly thought this was about the step instructions. My two previous answers here refer to the algorithmic step comments. I am sorry about that.

The test itself is fine, but I am curious what the numbers are for? I’m not opposed to a test for clicking a plain link.

Reply:
The numbers 123456 was a test David asked for. He wanted to test a link of numbers.

Reply:
Right, but that doesn’t answer my question _why_ they are significant.

Reply:
It seems to me that it is still just a string, so I don't know any other significance of a link with numbers. What do you want me to do with this test?

Reply:
If it isn’t special in any way, a string of `123456` is as good as anything else I guess.